### PR TITLE
feat: add create HA cluster e2e test spec

### LINKED ISSUE #87: Add create HA cluster e2e test spec
/area testing

### PR DESCRIPTION
**Pull request by sacredartr**
### What type of PR is this?
/kind flake

### What this PR does / why we need it:
add create HA cluster e2e test spec

### Which issue(s) this PR fixes:
Fixes #87 

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?
```
None
```

### Additional documentation, usage docs, etc.:
```docs

```

### CODE REVIEW

File: test/e2e/cluster/cluster_ha.go [hunk 1]
```diff
@@ -1,28 +1,124 @@
...
+		ginkgo.By("Check that there are enough available nodes")
+		nodes, err := f.Client.ListNodes(context.TODO(), kc.Queries{
+			Pagination:    query.NoPagination(),
+			LabelSelector: "!kubeclipper.io/nodeRole",
+		})
+		framework.ExpectNoError(err)
+		if len(nodes.Items) < 2 {
+			framework.Failf("Not enough nodes to test")
+		}
+		nodeList = corev1.WorkerNodeList{
```
Comment (Xvv-v) [inline test/e2e/cluster/cluster_ha.go:44]: msatre node can't be even number

**Review by Xvv-v — APPROVED** [review]

**Review by x893675 — APPROVED** [review]

**Pull request merged**